sympy/printing/latex.py

```diff
@@ -1076,7 +1076,12 @@ def _print_ceiling(self, expr, exp=None):
 
     def _print_log(self, expr, exp=None):
         if not self._settings["ln_notation"]:
-            tex = r"\log{\left(%s \right)}" % self._print(expr.args[0])
+            if len(expr.args) == 2:
+                argument = self._print(expr.args[0])
+                base = self._print(expr.args[1])
+                tex = r"\log_%s{\left(%s \right)}" % (base, argument)
```

Comment:
Do  you need brackets around the base in case it is `10` or some other value with more than 1 character? `log_{%s}...`

Reply:
There is additional complexity about unicode combining characters. 
For example, `len("ā")` is 2 character but displays as one character and it works fine without braces. So I'm not sure about doing that